scripts/test_memory_spikes.py

```diff
@@ -0,0 +1,128 @@
+# Python script to check if there are memory spikes when running out of 
+# order random inserts to TimescaleDB database
+import psutil
+import time
+import sys
+
+MEMORY_CAP = 300 * 1024 * 1024 # absolute memory cap
```

Comment:
Should these two variables be (optional) arguments to the program?

Reply:
We could. Don't feel strongly either way.